README.md

```diff
@@ -140,19 +150,23 @@ This project uses dotnet and docker to run the local database.
 # Interacting with the Client API
 ## Sending VRDR Records
 ### Submission Records
-1. Create a FHIR Record. The standard that specifies this format can be found [here](https://build.fhir.org/ig/HL7/vrdr/branches/Sep_2021_Connectathon/). There are also two public library implementations available to assist in the creation of FHIR Records, [VRDR-dotnet](https://github.com/nightingaleproject/vrdr-dotnet) and [VRDR_javalib](https://github.com/MortalityReporting/VRDR_javalib).
-2. Submit the record using a POST request to the `/record/submission` endpoint. The following example demonstrates how to make the request using [curl](https://curl.se/):
+1. Create a FHIR Record. The standard that specifies this format can be found at following depending on type of records
+	[VRDR](https://nightingaleproject.github.io/vital_records_fhir_messaging_ig/v2.0.0/onnectathon/).
+		There are also two public library implementations available to assist in the creation of FHIR Records, [VRDR-dotnet](https://github.com/nightingaleproject/vrdr-dotnet) and [VRDR_javalib](https://github.com/MortalityReporting/VRDR_javalib).
```

Comment:
add [BFDR-dotnet] too with a link to here https://github.com/nightingaleproject/vital-records-dotnet/tree/main/projects/BFDR